app/src/main/java/impacta/contactless/features/settings/SettingsScreen.kt

```diff
@@ -11,7 +24,40 @@ import impacta.contactless.ui.theme.KeyzTheme
 fun SettingsScreen(
     navController: NavController? = null,
 ) {
-    Text("Settings")
+    val iconColor = Color(0XFF42493F)
+//    val textColor = Color(0xFF1A1C19)
+    val textColor = Color(0xFFFFFFFF)
+    val alertColor = Color(0xFF93000A)
```

Comment:
Tenta referenciar as cores do tema o tanto quanto possível, se for mudar um tipo de cor depois, fica centralizado